### ISSUE TITLE
Can delivery bots be open source? I want it, it feels very interesting。thanks

### ISSUE THREAD

**Issue by z1006670445** [berichan/SysBot.ACNHOrders#35]

**Comment by berichan** [comment]
Sadly it isn't up to me to open source a codebase that isn't entirely my own.

**Comment by z1006670445** [comment]
> Sadly it isn't up to me to open source a codebase that isn't entirely my own.

Thank you for your reply. Can you share it with me separately?

**Comment by z1006670445** [comment]
> Sadly it isn't up to me to open source a codebase that isn't entirely my own.

If it's not convenient for you to provide it, could you please tell me how to obtain the deliver bot source code? thanks

**Comment by berichan** [comment]
> > Sadly it isn't up to me to open source a codebase that isn't entirely my own.
> 
> If it's not convenient for you to provide it, could you please tell me how to obtain the deliver bot source code? thanks

No, as above, the source is not mine to distribute and is unrelated to this repository.
